<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>talletaRahaa</class>
 <widget class="QDialog" name="talletaRahaa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Otsikko">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>841</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PANOAUTOMAATTI,  RAHAN TALLETUS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRahat">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSyotaRahat">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Syötä talletettavat rahat:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTalleta">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Talleta rahat tilille</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIlmoitukset">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTakaisin">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
